### PR TITLE
[DEST] add readme file for dest

### PR DESCRIPTION
**Pull request by zhby99**
Add readme file for dest

### CODE REVIEW

File: configs/dest/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# DEST
+
+[DEST: Depth Estimation with Simplified Transformer](https://arxiv.org/abs/2204.13791)
+
+## Introduction
+
+<!-- [ALGORITHM] -->
+
+<a href="https://github.com/NVIDIA/DL4AGX/tree/master/DEST">Official Repo</a>
```
Comment (ghost) [inline configs/dest/README.md:9]: Add something like "DEST was developed for dense estimation tasks such as depth estimation and semantic segmentation. The core of the method is the simplified attention that has achieved better accuracy and latency as compared to the standard attention mechanism". Please find more information the official repo (hpyerlink to repo).

File: configs/dest/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+## Abstract
+
+<!-- [ABSTRACT] -->
+
+Transformer and its variants have shown state-of-the-art results in many vision tasks recently, ranging from image classification to dense prediction. Despite of their success, limited work has been reported on improving the model efficiency for deployment in latency-critical applications, such as autonomous driving and robotic navigation. In this paper, we aim at improving upon the existing transformers in vision, and propose a method for self-supervised monocular Depth Estimation with Simplified Transformer (DEST), which is efficient and particularly suitable for deployment on GPU-based platforms. Through strategic design choices, our model leads to significant reduction in model size, complexity, as well as inference latency, while achieving superior accuracy as compared to state-of-theart. We also show that our design generalize well to other dense prediction task without bells and whistles.
```
Comment (ghost) [inline configs/dest/README.md]: add "...other prediction task such as semantic segmentation..."

**Review by ghost — COMMENTED** [review]
Add DEST as a bullet item to the top level read me

**Review by ghost — COMMENTED** [review]
Add license as segformer in https://github.com/open-mmlab/mmsegmentation/blob/master/LICENSES.md, our license is the same as segformer

File: configs/dest/README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+| DEST | MIT-B0   | 1024x1024 |  160000 |     - | -           | 64.34 | -         | - | - | - |
+| DEST | MIT-B1   | 1024x1024 |  160000 |     - | -            | 68.21 | -        | - | - | - |
+| DEST | MIT-B2   | 1024x1024 |  160000 |     - | -           | 71.89 | -         | - | - | - |
+| DEST | MIT-B3   | 1024x1024 |  160000 |    - | -           | 73.51 | -         | - | - | - |
+| DEST | MIT-B4   | 1024x1024 |  160000 |    - | -           | 73.99 | -         | - | - | - |
+| DEST | MIT-B5   | 1024x1024 |  160000 |    - | -          | 75.28 | -        | - | - | - |
+
+
+- Training of DEST is not very stable, which is sensitive to random seeds.
```
Comment (ghost) [inline configs/dest/README.md:51]: Add a sub heading of "Note:"
Add a note about training from scratch, not with a pretrained model
Add link to config, model/log

Comment (zhby99) [reply configs/dest/README.md:51]: model/logs are under: https://nvidia-my.sharepoint.com/:f:/p/boyinz/EqOEo1BZ1V9LpMKLoldH4YkBRvG4xWqhJSUdUVah7VhqDw?e=WB45AM

File: README.md [hunk 4]
```diff
@@ -168,6 +168,7 @@ Supported methods:
 - [x] [Segmenter (ICCV'2021)](configs/segmenter)
 - [x] [SegFormer (NeurIPS'2021)](configs/segformer)
 - [x] [K-Net (NeurIPS'2021)](configs/knet)
+- [x] [DEST (CVPR'2022)](configs/dest)
```
Comment (ghost) [inline README.md]: CVPRW, because this is a workshop paper

File: configs/dest/README.md (hunk 1 above)
Comment (ghost) [reply configs/dest/README.md:9]: Can you add this to introduction? The reason is that the paper's title is depth, and we want to divert from that narrow scope by explaining it's for dense prediction

File: configs/dest/README.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+<!-- [ALGORITHM] -->
+
+<a href="https://github.com/NVIDIA/DL4AGX/tree/master/DEST">Official Repo</a>
+
+## Abstract
+
+<!-- [ABSTRACT] -->
+
+Transformer and its variants have shown state-of-the-art results in many vision tasks recently, ranging from image classification to dense prediction. Despite of their success, limited work has been reported on improving the model efficiency for deployment in latency-critical applications, such as autonomous driving and robotic navigation. In this paper, we aim at improving upon the existing transformers in vision, and propose a method for self-supervised monocular Depth Estimation with Simplified Transformer (DEST), which is efficient and particularly suitable for deployment on GPU-based platforms. Through strategic design choices, our model leads to significant reduction in model size, complexity, as well as inference latency, while achieving superior accuracy as compared to state-of-theart. We also show that our design generalize well to other dense prediction task such as semantic segmentation without bells and whistles.
```
Comment (ghost) [inline configs/dest/README.md]: Replace with the following:

Transformer and its variants have shown state-of-the-art results in many vision tasks recently, ranging from image classification to dense prediction. Despite of their success, limited work has been reported on improving the model efficiency for deployment in latency-critical applications, such as autonomous driving and robotic navigation. In this paper, we aim at improving upon the existing transformers in vision, and propose a method for Dense Estimation with Simplified Transformer (DEST), which is efficient and particularly suitable for deployment on GPU-based platforms. Through strategic design choices, our model leads to significant reduction in model size, complexity, as well as inference latency, while achieving superior accuracy as compared to state-of-the-art in the task of self-supervised monocular depth estimation. We also show that our design generalize well to other dense prediction task such as semantic segmentation without bells and whistles.

File: configs/dest/README.md (hunk 3 above)
Comment (ghost) [reply configs/dest/README.md:51]: nvidia-my.sharepoint.com is not accessible from outside, can you move to your personal google drive or dropbox?

Comment (zhby99) [reply configs/dest/README.md:51]: https://drive.google.com/drive/folders/1MOnW41XKPRYseOtxh_wgsbgpt7xwmeLb?usp=sharing

File: configs/dest/README.md (hunk 5 above)
Comment (zhby99) [reply configs/dest/README.md]: addressed

File: README.md (hunk 4 above)
Comment (zhby99) [reply README.md]: addressed
